pkg/audit/newverifier_test.go

```diff
@@ -283,6 +287,7 @@ func TestDownloadSharesDialTimeout(t *testing.T) {
 // If this test fails, this most probably means we made a backward-incompatible
 // change that affects the audit service.
 func TestDownloadSharesDownloadTimeout(t *testing.T) {
+	t.Skip()
```

Comment:
should the audit tests be still skipped?

Reply:
My mistake. Reverted "skip" code.